src/components/number-picker/number-picker.component.ts

```diff
@@ -102,11 +102,20 @@ export class NumberPickerComponent implements ControlValueAccessor {
     /** Store the current valid state */
     _valid: boolean = true;
 
+    /* */
```

Comment:
Could we add a comment here describing the reason this is needed here? Just to make it obvious in future to any developer working on this component